pure_protobuf/protobuf.py

```diff
@@ -508,6 +512,7 @@ def __init__(self, message_type):
         """
         Initializes a new instance of the specified message type.
         """
+        super(Message, self).__init__()
```

Comment:
This can just be `super()` in Python 3

Reply:
I'm trying to keep compatibility with 2 & 3 for now